addons/mail/static/src/js/activity.js

```diff
@@ -47,6 +47,11 @@ const KanbanActivity = AbstractField.extend({
         this.defaultActivityType = options.activityType;
         this.isActivityViewCell = options.isActivityViewCell;
     },
+    async willStart() {
+        await this._super();
+        await owl.Component.env.services.messaging.modelManager.messagingCreatedPromise;
+        this.messaging = owl.Component.env.services.messaging.modelManager.messaging;
```

Comment:
why not just `await .get()`?

Reply:
get includes messaging initialized which is longer than necessary, just messaging created is enough here